### PR TITLE
Search page

### PR DESCRIPTION
**Pull request by dimas1976**
https://review-searchpage-4ayp5adgahqj.herokuapp.com/search

### CODE REVIEW

File: src/app/pages/Search/Search.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+    layout: 'fullscreen',
+  },
+};
+
+export const SearchPage = (): JSX.Element => <Search />;
```
Comment (marwinburesch) [inline src/app/pages/Search/Search.stories.tsx:16]: We decided, that we don't really need stories for our pages. Please remove the file

File: src/app/pages/Search/Search.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <div className={styles.searchbar}>
+          <SearchBar
+            searchResult={searchValue}
+            setSearchResult={setSearchValue}
+            handleSubmit={handleSubmit}
+          />
+        </div>
+
+        <TagGroup tagList={mockTagGroupProps} />
+        <p>Search Results(3)</p>
```
Comment (marwinburesch) [inline src/app/pages/Search/Search.tsx:31]: I'd separate `SearchBar`  from `TagGroup`

File: src/app/pages/Search/Search.module.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  color: var(--text-primary);
+  height: 100vh;
+}
+
+main {
+  overflow: auto;
+}
+
+.search {
+  width: 100vw;
```
Comment (marwinburesch) [inline src/app/pages/Search/Search.module.css:14]: Is there a way to not set this property anywhere on lower leves than `#app` or the main grid for this page?
We might have to set a `max-width` for `TagGroup`

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Excellent work, thank you! ☔

File: src/app/pages/Search/Search.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        searchValue={searchValue}
+        setSearchValue={setSearchValue}
+        handleSubmit={handleSubmit}
+      />
+
+      <TagGroup className={styles.tagGroup} tagList={mockTagGroupProps} />
+      <p className={styles.searchResult}>Search Results(3)</p>
+
+      <main className={styles.cards}>
+        {mockCards?.map((mockCard) => (
```
Comment (phlgr) [inline src/app/pages/Search/Search.tsx:32]:
```suggestion
        {mockCards && mockCards.map((mockCard) => (
```

File: src/app/pages/Search/Search.module.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+.searchbar {
+  margin: 1rem;
+}
+
+.searchResult {
+  padding-left: 1rem;
+}
+.cards {
```
Comment (phlgr) [inline src/app/pages/Search/Search.module.css:28]: whitespace missing

File: src/app/pages/Search/Search.module.css [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  grid-template-rows: auto auto auto auto 1fr auto;
+}
+
+.header {
+  padding: 1rem 0 0 0.7rem;
+}
+
+main {
+  overflow: auto;
+}
```
Comment (phlgr) [inline src/app/pages/Search/Search.module.css:15]: You could move this property to the `.cards` class

**Review by phlgr — CHANGES_REQUESTED** [review]

**Review by phlgr — APPROVED** [review]

**Review by marwinburesch — APPROVED** [review]

**Pull request merged**